save_coefs.py

```diff
@@ -0,0 +1,52 @@
+from main import DenoisingAutoEncoder
+import numpy as np
+from skimage import measure
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d.art3d import Poly3DCollection
+import torch
+from torch import FloatTensor
+import open3d as o3d
+import os
+import random
+import json
+from tqdm import tqdm
+
+
+M = np.array([[0.80656762, -0.5868724, -0.07091862],
+              [0.3770505, 0.418344, 0.82632997],
+              [-0.45528188, -0.6932309, 0.55870326]])
+
+
+def PointCloud(points, colors=None):
+    pcd = o3d.geometry.PointCloud()
+    pcd.points = o3d.utility.Vector3dVector(points)
+    if colors is not None:
+        pcd.colors = o3d.utility.Vector3dVector(colors)
+    return pcd
+
+
+ckpt = torch.load("epoch=60.ckpt", map_location=torch.device("cuda"))
+hparams = ckpt.get("hparams")
+state_dict = ckpt.get("state_dict")
+model = DenoisingAutoEncoder(hparams=hparams, train_set=(
+    "s2_train.npz", "filled_dense"), val_set=("s2_test.npz", "filled_dense"))
+model.load_state_dict(state_dict)
+# voxels = np.load("s2_test.npz")["filled_dense"]
+# voxels = np.load("s2_train.npz")["filled_dense"]
+model.cuda()
+root_path = 'D:/data/pix3d/'
+with open(root_path + 'pix3d.json') as json_path:
```

Comment:
Would you like to change these lines into a `Dataloader` manner?

Reply:
Also will be modified in the next version after my midterm.